release/files_common

```diff
@@ -187,7 +187,7 @@ selfdrive/controls/lib/longitudinal_mpc_lib/.gitignore
 selfdrive/controls/lib/lateral_mpc_lib/*
 selfdrive/controls/lib/longitudinal_mpc_lib/*
 
-selfdrive/hardware
+system/hardware
```

Comment:
`selfdrive/hardware` is a symlink we need to keep around. It's used by the updater, so it needs to stay around or a bit to stay backwards compatible with older versions updating to the latest version.